packages/liveblocks-client/src/index.ts

```diff
@@ -5,6 +5,7 @@ import { PKG_FORMAT, PKG_NAME, PKG_VERSION } from "./version";
 detectDupes(PKG_NAME, PKG_VERSION, PKG_FORMAT);
 
 export type {
+  Awaitable,
```

Comment:
I've also decided to re-export this in `@liveblocks/client`, because this type is part of our public API already (in some return types).